pages/common/gyb.md

```diff
@@ -0,0 +1,20 @@
+# Got Your Back - GYB
+
+> Command line tool for backing up your Gmail messages to your computer using Gmail's API over HTTPS.
+> More information: <https://github.com/GAM-team/got-your-back>.
+
+- Estimate the number and the size of all emails on your Gmail account:
+
+`gyb --email {{youremail@gmail.com}} --action estimate`
```

Comment:
This makes it a little more readable. Also, do you have to enter your Google credentials for this?
```suggestion
`gyb --email {{email@gmail.com}} --action estimate`
```

Reply:
Yes. The tool works only for GMail